test/manual/amp-mega-menu.amp.html

```diff
@@ -0,0 +1,138 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>amp-mega-menu</title>
+  <link rel="canonical" href="amps.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    #title {
+      text-align: center;
+      font-size: 25px;
+      padding: 15px;
+    }
+    header > * {
+      background: white;
+    }
+    .heading, .content, .section {
+      padding: 10px;
+      margin: 0;
+    }
+    .section {
+      display: inline-block;
+      vertical-align: top;
+    }
+    ul {
+      list-style-type: none;
+    }
+    amp-embed {
+      float: right;
+      margin-right: 50px;
+    }
+    fieldset {
+      border: none;
+    }
+    #main {
+      background: #ccc;
+      font-size: 18px;
+    }
+  </style>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-mega-menu" src="https://cdn.ampproject.org/v0/amp-mega-menu-0.1.js"></script>
+  <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+  <script async custom-element="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
+  <script async custom-element="amp-sidebar" src="https://cdn.ampproject.org/v0/amp-sidebar-0.1.js"></script>
+  <script async custom-element="amp-accordion" src="https://cdn.ampproject.org/v0/amp-accordion-0.1.js"></script>
+  <script async custom-element="amp-ad" src="https://cdn.ampproject.org/v0/amp-ad-0.1.js"></script>
+  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+  <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+</head>
+<body>
+  <header>
+    <div id="title">amp-mega-menu</div>
+
+    <amp-mega-menu layout="fixed-height" height="55">
+      <amp-list layout="fixed-height" height="600" src="/list/ecommerce-nested-menu" items="." single-item>
```

Comment:
Having the `amp-list` be of `fixed-height` means that elements under it won't be clickable at all (whether the mask is open or not). You can see on your demo that the `Visit AMP` link under the menu is not clickable because of this. We need to find a way around this (either toggle `visibility` on this element (not good since `amp-list` is not part of `amp-mega-menu`'s expected hierarchy), have the `amp-mega-menu` be `overflow: hidden` unless open, or something else.

Reply:
That's a great observation (and a major oversight on my part). Keeping `overflow: hidden` unless the menu is open seems to be a suitable solution.